app/controllers/api/rooms_controller.rb

```diff
@@ -1,12 +1,10 @@
 class Api::RoomsController < Api::BaseController
 
   def index
-    rooms = Room.where(conference_id: params[:conference_id]).page params[:page]
-    puts rooms.first.id
+    rooms = Room.where(conference_id: params[:conference_id])
```

Comment:
To save us some association queries, let's do this:
`rooms = Room..includes(:conference).where conference_id: params[:conference_id]`.